src/sentry/models/artifactbundle.py

```diff
@@ -56,14 +56,15 @@ class ReleaseArtifactBundle(Model):
     organization_id = BoundedBigIntegerField(db_index=True)
     release_name = models.CharField(max_length=250)
     dist_id = BoundedBigIntegerField(null=True)
```

Comment:
Why not kill this in the same PR?

Reply:
https://develop.sentry.dev/database-migrations/#columns

Reply:
Technically we could remove this column, iff we don't have the new upload endpoint called in production during the deployment. Because otherwise we will try to insert data with the old model in the new table.

Reply:
I would agree but this model should not be used in production, so the change should be safe.

Reply:
How do I skip the change then? The CI was throwing an error.